src/zenml/integrations/kubernetes/orchestrators/manifest_utils.py

```diff
@@ -47,73 +119,76 @@ def build_pod_manifest(
             Can be used to assign certain roles to a pod, e.g., to allow it to
             run Kubernetes commands from within the cluster.
         env: Environment variables to set.
+        mount_local_stores: Whether to mount the local stores path inside the
+            pod.
 
     Returns:
         Pod manifest.
     """
     env = env.copy() if env else {}
     env.setdefault(ENV_ZENML_ENABLE_REPO_INIT_WARNINGS, "False")
 
-    spec: Dict[str, Any] = {
-        "restartPolicy": "Never",
-        "containers": [
-            {
-                "name": "main",
-                "image": image_name,
-                "command": command,
-                "args": args,
-                "env": [
-                    {"name": name, "value": value}
-                    for name, value in env.items()
-                ],
-            }
+    container_spec = k8s_client.V1Container(
```

Comment:
Nice that you cleaned this up to use the proper k8s classes instead of the dicts I passed around back then 🙏